packages/flutter/test/material/drawer_test.dart

```diff
@@ -479,19 +479,9 @@ void main() {
     await tester.pumpWidget(
       MaterialApp(
         home: OrientationBuilder(
-          builder: (BuildContext context, Orientation orientation) {
-            switch (orientation) {
-              case Orientation.portrait:
-                return Scaffold(
-                  drawer: const Text('drawer'),
-                  body: Container(),
-                );
-              case Orientation.landscape:
-                return Scaffold(
-                  appBar: AppBar(),
-                  body: Container(),
-                );
-            }
+          builder: (BuildContext context, Orientation orientation) => switch (orientation) {
+            Orientation.portrait  => Scaffold(body: Container(), drawer: const Text('drawer')),
+            Orientation.landscape => Scaffold(body: Container(), appBar: AppBar()),
```

Comment:
This was slightly nicer when the arguments to Scaffold were each on their own line, which is more Flutter-style.

Reply:
(here and below)